src/components/cardbuilder/card.scss

```diff
@@ -834,6 +833,7 @@ button::-moz-focus-inner {
 }
 
 .card-hoverable:hover .cardOverlayContainer {
+    cursor: pointer;
     opacity: 1;
 }
```

Comment:
There may be no overlay: https://github.com/jellyfin/jellyfin-web/blob/c0fb9eca82bee9bda8f117d1e3ef38b21ad27721/src/components/cardbuilder/cardBuilder.js#L1206-L1208

Could you check this in TV layout?

Reply:
Hi, sorry still new to this so this might sound stupid - What would be the best way to test the TV layout?

Reply:
This seems to work fine - Unless I am missing something.